common/crypto_utils/avalon_crypto_utils/worker_signing.py

```diff
@@ -0,0 +1,299 @@
+
+        decoded_signature = crypto_utility.base64_to_byte_array(signature)
+
+        verify = self.verify_signature_from_pubkey(decoded_signature,
+                                                   final_hash,
+                                                   verification_key)
+        return verify
+
+# -----------------------------------------------------------------------------
+    def verify_create_receipt_signature(self, input_json):
+        """
+        Function to verify the signature of work order receipt create
+        Parameters:
+            - input_json is dictionary contains request payload of
+              WorkOrderReceiptRetrieve API as define EEA spec 7.2.2
+        Returns enum type SignatureStatus
+        """
+        input_json_params = input_json['params']
+
+        concat_string = input_json_params["workOrderId"] + \
+            input_json_params["workerServiceId"] + \
+            input_json_params["workerId"] + \
+            input_json_params["requesterId"] + \
+            str(input_json_params["receiptCreateStatus"]) + \
+            input_json_params["workOrderRequestHash"] + \
+            input_json_params["requesterGeneratedNonce"]
+        concat_hash = bytes(concat_string, "UTF-8")
+        final_hash = bytes(crypto_utility.compute_message_hash(concat_hash))
+        signature = input_json_params["requesterSignature"]
+        verification_key = \
+            input_json_params["receiptVerificationKey"].encode("ascii")
+
+        decoded_signature = crypto_utility.base64_to_byte_array(signature)
+        verify = self.verify_signature_from_pubkey(decoded_signature,
+                                                   final_hash,
+                                                   verification_key)
+        return verify
+
+# -----------------------------------------------------------------------------
+    def verify_encryption_key_signature(
+            self, encryption_key_signature, encryption_key, verifying_key):
+        """
+        Utils function to verify integrity of worker encryption key using
+        worker verification key
+        @params encryption_key_signature - Signature computed on hash
+                                           of encryption key
+        @params encryption_key - Public encryption key of the worker
+        @params verifying_key - Public signing key or verification key
+                                of the worker
+        returns SignatureStatus.PASSED in case of successful verification
+                SignatureStatus.FAILED in case of verification failure
+        """
+
+        encrypt_key_sig_bytes = hex_to_byte_array(encryption_key_signature)
+        encrypt_key_bytes = crypto_utility.string_to_byte_array(encryption_key)
+        encryption_key_hash = worker_hash.WorkerHash().compute_message_hash(
+            encrypt_key_bytes)
+
+        verify = self.verify_signature_from_pubkey(encrypt_key_sig_bytes,
```

Comment:
Does this not flag a lint or lgtm ? you could have simply returned the function call. Intermediate variable not needed.

Reply:
Apparently it didn't flag. Agree that its better to return the function call. Made the change wherever necessary in this file.